packages/syft/src/syft/service/project/project.py

```diff
@@ -501,6 +505,45 @@ def orchestrate_enclave_execution(self) -> Any:
         self.request_asset_transfer()
         return self.request_execution()
 
+    def view_attestation_report(self, return_report: bool = False) -> dict | None:
```

Comment:
MAYBE: shall we include a field that also allows us to get gpu attestation, or shall we keep gpu attestation to a follow up PR?

Reply:
Added it in this PR